Microsoft.Dynamics365.UIAutomation.Sample/app.build.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Es is true by default  
TestSettings:27:  PrivateMode = UsePrivateMode ?? true,

Reply:
I went ahead and approved/merged to avoid holding this up. 

I think this is marked on the wrong comment so I might be misreading, but the standalone app.config file has UsePrivateMode=False. Does the app.build.config always override that? If not, then testing standalone in VS will default to not using private mode?

Reply:
Sorry. I was testing & forgot to put in back to true in app.config. If you can fix it, directly in develop, please do it. I think we should update the documentation too.
The DevOps task: FileTransform@1 will replace the app.config values with the values in app.build.config, that in this particular configuration are being replaced by the Variable Values defined in Pipeline. VS will not replace it by default.
Thanks